src/components/LetterLoad.vue

```diff
@@ -69,6 +76,10 @@ export default {
     currentDate () {
       const current = new Date()
       const date = `${current.getDate()}/${current.getMonth() + 1}/${current.getFullYear()}`; return date
+    },
+    persistCustomization(e){
+      let customText = e.target.value
+      this.$store.dispatch('setLetterAttrs', {userCustomization:customText})
```

Comment:
To update a single piece of state, use the setGenericValue mutation. setLetterAttrs expects a more complex object and will override the current state.

`this.$store.commit('setGenericValue', { key: 'userCustomization', value: this.customText })`